data/strings/FreeColMessages.properties

```diff
@@ -1797,7 +1797,7 @@ model.foundingFather.juanDeSepulveda.text=A Spanish theologian who spoke out for
 model.foundingFather.juanDeSepulveda.birthAndDeath=1781-1872
 
 model.foundingFather.bartolomeDeLasCasas.name=Bartolome de las Casas
-model.foundingFather.bartolomeDeLasCasas.description=All existing Indian converts are converted to free colonists.
+model.foundingFather.bartolomeDeLasCasas.description=All existing Indian converts can be converted to free colonists or educated in some outdoor work.
```

Comment:
This changes the text also when playing with the classic ruleset ... and that would be wrong. The new text should be placed in a new file called "ModMessages.properties" in the "data/rules/freecol" directory.

I have noticed that we do not have support for including mod translatable messages to the "FreeColMessages.properties". This will be necessary in order to utilize translatewiki for getting the strings translated. We can, for example, automatically add/update the strings using the prefix "mod.MODID." as a part of our build process.

Reply:
I will send an email about supporting translations of rules/mods/maps/scenarios on the developer's mailing list.